### PR TITLE
New: Added use helper (fixes #504)

### LINKED ISSUE #504: Add model selection helper
### Subject of the issue

Allow feedback from other questions to be display out of place.

https://github.com/adaptlearning/adapt-contrib-core/blob/8b36e3ee68941b62e711198f49fa9345c1758132/js/models/questionModel.js#L311-L313

```hbs
{{#use 'c-200'}}{{{feedbackMessage}}}{{/use}}
```

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes #504

### New
* Added use helper to allow ```{{#use 'c-200'}}{{{feedbackMessage}}}{{/use}}```

### CODE REVIEW

**Comment by swashbuck** [comment]
@oliverfoster This doesn't seem to work unless `data` is imported at the top of _[js/helpers.js](https://github.com/adaptlearning/adapt-contrib-core/blob/master/js/helpers.js)_.

``` Javascript
import data from 'core/js/data';
```

File: js/helpers.js [hunk 1]
```diff
@@ -67,6 +67,14 @@ const helpers = {
     }
   },
 
+  use(id, block) {
+    if (!block.fn) return;
+    try {
+      const model = data.findById(id);
```
Comment (swashbuck) [inline js/helpers.js:73]: Getting `ReferenceError: data is not defined` here. See my previous comment.

**Review by swashbuck — APPROVED** [review]
👍

**Review by joe-allen-89 — APPROVED** [review]
👍

**Pull request merged**